weldx/tests/test_transformations.py

```diff
@@ -851,17 +851,17 @@ def test_reset_reference_time_exceptions(
         [
             (  # broadcast left
                 TS("2020-02-10"),
-                TDI([1, 2], "D"),
+                TDI([1, 2, 10], "D"),
                 TS("2020-02-10"),
-                r_mat_z([0, 0]),
-                np.array([[2, 8, 7], [2, 8, 7]]),
+                r_mat_z([0, 0, 0]),
+                np.array([[2, 8, 7], [2, 8, 7], [2, 8, 7]]),
             ),
             (  # broadcast right
                 TS("2020-02-10"),
-                TDI([29, 30], "D"),
+                TDI([22, 29, 30], "D"),
                 TS("2020-02-10"),
-                r_mat_z([0.5, 0.5]),
-                np.array([[3, 1, 2], [3, 1, 2]]),
+                r_mat_z([0.5, 0.5, 0.5]),
+                np.array([[3, 1, 2], [3, 1, 2], [3, 1, 2]]),
```

Comment:
I guess, that it does make sense to collapse edge cases (the only time intersection is at the boundaries) to a static system too since we still have always the same value

Reply:
Implemented this behavior now.